samples/benchmarks/coremark/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include "app_aliases_common.overlay"
+
+/* The following configuration is required to run the CPUFLPR core.
+ * It is imported from the nordic-flpr snippet.
```

Comment:
Can we simply use the snippet from Zephyr and avoid copying file content here?
Ref: https://docs.zephyrproject.org/latest/build/snippets/using.html#application-required-snippets

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-31977